release-notes/6.0/preview/api-diff/preview1/.Net/6.0-preview1_System.Diagnostics.md

```diff
@@ -0,0 +1,18 @@
+# System.Diagnostics
+
+``` diff
+ namespace System.Diagnostics {
+     public class Activity : IDisposable {
++        public Activity SetBaggage(string key, string value);
+     }
+     public sealed class ActivitySource : IDisposable {
++        public Activity StartActivity(ActivityKind kind, ActivityContext parentContext = default(ActivityContext), IEnumerable<KeyValuePair<string, object>> tags = null, IEnumerable<ActivityLink> links = null, DateTimeOffset startTime = default(DateTimeOffset), string name = "");
+-        public Activity StartActivity(string name, ActivityKind kind = ActivityKind.Internal);
++        public Activity StartActivity(string name = "", ActivityKind kind = ActivityKind.Internal);
```

Comment:
Just wondering, does it matter if the parameter attributes are not showing here?

Reply:
Its a limitation with the tool.